### PR TITLE
Google Maps Places Plugin

### PR DESCRIPTION
**Pull request by TimoPieti**
Migrated existing plugin code to use newest Flutter.

### CODE REVIEW

File: google_maps_places/LICENSE [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Copyright 2013 The Flutter Authors. All rights reserved.
```
Comment (jokerttu) [inline google_maps_places/LICENSE:1]: This might change, depending of the target repository

Comment (TimoPieti) [reply google_maps_places/LICENSE:1]: Yeah, but than can be changed by search and replace. But maybe we should add some description to root level README.md that this mimics flutter/packages plugin structure and format.

File: google_maps_places/pubspec.yaml [hunk 2]
```diff
@@ -5,8 +5,8 @@ homepage:
 publish_to: none
 
 environment:
-  sdk: ">=2.18.4 <3.0.0"
-  flutter: ">=2.5.0"
+  sdk: ">=2.18.0 <4.0.0"
+  flutter: ">=3.3.0"
```
Comment (jokerttu) [inline google_maps_places/pubspec.yaml:9]: Not sure why we do now allow older flutter versions here?
This did not work with flutter >=2.5.0?

File: google_maps_places/LICENSE (hunk 1 above)
Comment (jokerttu) [reply google_maps_places/LICENSE:1]: @TimoPieti yes, also we should maybe write open the folder structure in the README

File: google_maps_places/pubspec.yaml (hunk 2 above)
Comment (TimoPieti) [reply google_maps_places/pubspec.yaml:9]: I copied these dependies values from Google Maps flutter package.

**Comment by jokerttu** [comment]
LGTM

**Pull request merged**